src/modules/crypto/decrypt.ts

```diff
@@ -39,17 +63,27 @@ export const decrypt = async (encryptedAsBase64: string, symmetricKeyGetter: Sym
             let symmetricKeyPromise = symmetricKeyGetter.derivates.get(derivationMethodBytes);
             if (!symmetricKeyPromise) {
                 winston.debug(`Computing new derivate with method: ${encryptedData.keyDerivation.algo}`);
-                symmetricKeyPromise = getDerivateUsingParametersFromEncryptedData(
-                    symmetricKeyGetter.secrets.masterPassword,
-                    encryptedData
-                );
+                if (encryptedData.keyDerivation.algo === 'noderivation') {
```

Comment:
why don't you treat this case in `getDerivateUsingParametersFromEncryptedData`

Reply:
sounds obvious now that you say it